code/datums/tips/tips.dm

```diff
@@ -0,0 +1,341 @@
+		"As the AI, you can take pictures with your camera and upload them to newscasters, or print them out on request.",
+		"As the AI, you can check the status of any stationbounds on the robotics console in your core, in the Reasearch Director's office.",
+		"As a Cyborg, choose your module carefully, as only a roboticist can let you repick it. Remember that you don't need to choose immediately after you spawn!",
+		"As a Cyborg, you are immune to most forms of stunning, and excel at almost everything far better than humans. However, flashes can easily make you blind and you cannot do any precision work as you lack hands.",
+		"As a Cyborg, you are almost impervious to fires and heat. If you are rogue, you can release phoron fires everywhere and walk through them without a care in the world, if you can handle the bwoink afterwards.",
+		"As a Cyborg, you are extremely vulnerable to EMPs as EMPs both stun you and damage you. Ion rifles or a traitor with an EMP kit can kill you in seconds.",
+		"As a Maintenance Drone, you can put on hats, but not take them off.",
+		"As a Maintenance Drone, you cannot repair yourself but you can ask other maintenance drones to do it for you."
+	)
+
+
+/*
+	SPECIES TIPS
+*/
+
+/datum/tip/species
+	category = CATEGORY_SPECIES
+	messages = list(
+		"Unathi are the only species capable of wearing the rare Breacher hardsuits.",
+		"As an Unathi, you can devour small mobs after some time.",
+		"As a Tajara, you move pretty fast. Zoom zoom, kitty.",
+		"As a Tajara, your resistance to cold probably doesn't actually help you in space. Feel free to try, though.",
+		"As a Skrell, you can look pretty and...uhh...not slip?",
+		"As a Skrell, you have free reign to validhunt synthetics. (Not really, please don't do this oh God what have I done).",
+		"As a Human, you are the best. Why do you care about your mechanics?",
+		"As a Human, you are really very great.",
+		"Due to IPCs' synthetic nature, they're immune to most chemicals and gasses.",
+		"IPCs can survive longer than most species in space, despite their supposed \"weakness\".",
+		"As a Diona, you can survive pretty much anything.",
+		"Dionae die in darkness. Find the light at the end of the tunnel, and quick.",
+		"All Vaurca can remotely speak to any other Vaurca on board. Not that there are any.",
+		"Vaurca can wade freely through phoron gas. Avoid phoron fires though. Somehow a species that breathes phoron gas is really weak to fire.",
+		"Despite the name, Auto-Hiss can be used for Unathi, Tajara, and Vaurca.",
+		"As a Diona, you can store items in your stomach, including guns."
+	)
+
+
+/*
+	GHOST SPAWNER TIPS
+*/
+
+/datum/tip/ghostspawn
+	category = CATEGORY_GHOSTSPAWN
+	messages = list(
+		"As an ERT trooper, your most powerful weapons are your team mates. Your second most powerful weapon is your bigass gun.",
```

Comment:
This whole section should probably be rewritten, as the most likely ghostspawn is going to be an off-map role, and not an ERT.